vmdb/app/models/metric/ci_mixin/capture/openstack_base.rb

```diff
@@ -37,134 +37,355 @@ def perf_release_openstack
   end
 
   def perf_capture_data_openstack_base(metric_capture_module, start_time, end_time, resource_filter, metadata_filter)
-    # some meters can get gathered by directly querying for the "resource_id",
-    #   but other meters can only be gathered by examining the
-    #   "metadata.instance_id"
     log_header = "MIQ(#{self.class.name}.perf_collect_data_openstack) [#{start_time} - #{end_time}]"
-    $log.debug "#{log_header} start_time: #{start_time}, end_time: #{end_time}"
+    $log.debug "#{log_header} id:[#{name}] start_time: #{start_time}, end_time: #{end_time}"
```

Comment:
Right, putting there perf_capture_data_openstack_base, the warn log contains list of counters, which are unique to each provider, so that should be enough.